src/BuildingRegistry.Projections.Wms/BuildingV2/BuildingV2Projections.cs

```diff
@@ -168,6 +168,38 @@ public BuildingV2Projections(WKBReader wkbReader)
                 var item = await context.BuildingsV2.FindAsync(message.Message.BuildingPersistentLocalId, cancellationToken: ct);
                 item.Version = message.Message.Provenance.Timestamp;
             });
+
+            When<Envelope<BuildingMergerWasRealized>>(async (context, message, ct) =>
+            {
+                var item = new BuildingV2
+                {
+                    PersistentLocalId = message.Message.BuildingPersistentLocalId,
+                    Id = PersistentLocalIdHelper.CreateBuildingId(message.Message.BuildingPersistentLocalId),
+                    Status = BuildingStatus.Realized,
+                };
+                SetGeometry(item, message.Message.ExtendedWkbGeometry, MeasuredByGrbMethod);
+                item.Version = message.Message.Provenance.Timestamp;
```

Comment:
Version can be set when creating the item object (so below Status = BuildingStatus.Realized)